content/F2014/F14-media-sarah-sharp.md

```diff
@@ -250,226 +249,221 @@ read from the USB devices.  And they do that through an interface to the Linux
 kernel.
 
 So USBFS is basically that interface.  It is a bunch of system calls.  The
-system calls, types it uses is actually Ioctls. And ioctls are like this really
+type of system calls that it uses is actually Ioctls. Ioctls are like this really
 weird system call that are like, if you can't find any other system call that
-fits, then use this one.  Basically you just pass a blob down to it and it gets
-munged and people fix it in the kernel and decode it.  So basically USBFS 2,
+fits, then you use this one.  Basically you just pass a blob down to it and it gets
+munged and people fix it in the kernel and decode it.  For USBFS 2,
 the goal of that was to replace these ioctl calls which were kind of ugly with
```

Comment:
I would probably run a search and replace to change all instances of `ioctl` to ioctl surrounded by backticks "`".